include/queue/queue.h

```diff
@@ -10,14 +10,16 @@
 #define DATA_SIZE 8
 
 typedef struct {
-	uint8_t size;
-    uint8_t index;
-    uint8_t content[MAX_SIZE][DATA_SIZE];
+  uint8_t head;
```

Comment:
Why is a head and a tail required?

Reply:
If you're building a circular buffer, it would be best to rename the structure as a `cbuffer` or `circular_buffer`

Reply:
minimal complexity

Reply:
I'll make a second enqueue function for circular or regular functionality